### PR TITLE
GRL - camera facing stuff

### PR DESCRIPTION
**Pull request by RolandCsibrei**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by RolandCsibrei** [comment]
- [ ] Create GreasedLineBaseMesh 
- [ ] Add simple material support for non camera facing lines
- [ ] Dont't create widths buffer if there are no user defined widths

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14184/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14184/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14184/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14184/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14184/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14184/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14184/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14184/merge#BCU1XR#0

File: packages/dev/core/src/Materials/greasedLineSimpleMaterial.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import type { BaseTexture } from "./Textures/baseTexture";
 
 import "../Shaders/greasedLine.fragment";
 import "../Shaders/greasedLine.vertex";
-import "../Shaders/greasedLineNonCameraFacing.vertex";
+// import "../Shaders/greasedLineNonCameraFacing.vertex";
```
Comment (Popov72) [inline packages/dev/core/src/Materials/greasedLineSimpleMaterial.ts:15]:
```suggestion
```

File: packages/dev/core/src/Meshes/greasedLineRibbonMesh.ts [hunk 2]
```diff
@@ -313,7 +327,7 @@ export class GreasedLineRibbonMesh extends Mesh {
         //         this._widths.push(w);
         //     }
         // } else {
-            this._widths = widths;
+        this._widths = widths;
```
Comment (Popov72) [inline packages/dev/core/src/Meshes/greasedLineRibbonMesh.ts:330]: nit: You should remove the coments before and after this line.

File: packages/dev/core/src/Materials/greasedLineSimpleMaterial.ts (hunk 1 above)
Comment (RolandCsibrei) [reply packages/dev/core/src/Materials/greasedLineSimpleMaterial.ts:15]: Hello, this is just a draft PR. Thanks anyway.

Comment (Popov72) [reply packages/dev/core/src/Materials/greasedLineSimpleMaterial.ts:15]: Sorry, will wait for your "go" once it's ok for you before reviewing!

**Pull request closed without merging**